### PR TITLE
[Docs] update the document format

### PR DESCRIPTION
**Pull request by luzhijing**
update the document format

## Proposed changes

Close related #issue (replace it with issue number if it exists).

Describe the overview of changes, and introduce why we need it.

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)
- [ ] Code refactor (Modify the code structure, format the code, etc...)
- [ ] Optimization. Including functional usability improvements and performance improvements.
- [ ] Dependency. Such as changes related to third-party components.
- [ ] Other.

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have created an issue on (Fix #ISSUE) and described the bug/feature there in detail
- [ ] Compiling and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If these changes need document changes, I have updated the document
- [x] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at dev@doris.apache.org by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by hf200012 — DISMISSED** [review]
LGTM

File: docs/zh-CN/administrator-guide/export-manual.md [hunk 1]
```diff
@@ -125,7 +125,7 @@ WITH BROKER "hdfs"
 
 * `label`：本次导出作业的标识。后续可以使用这个标识查看作业状态。
 * `column_separator`：列分隔符。默认为 `\t`。支持不可见字符，比如 '\x07'。
-* columns：要导出的列，使用英文状态逗号隔开，如果不填这个参数默认是导出表的所有列
+* `columns`：要导出的列，使用英文状态逗号隔开，如果不填这个参数默认是导出表的所有列
```
Comment (jackwener) [inline docs/zh-CN/administrator-guide/export-manual.md:128]:
```suggestion
* `columns`：要导出的列，使用英文状态逗号隔开，如果不填这个参数默认是导出表的所有列。
```

**Review by jackwener — COMMENTED** [review]
a tiny correction. it will be ok to just apply it.

**Comment by luzhijing** [comment]
> a tiny correction. it will be ok to just apply it.

ok,thank you!

**Review by HappenLee — APPROVED** [review]
LGTM

**Pull request merged**